mslice/plotting/plot_window/plot_figure.py

```diff
@@ -89,16 +99,21 @@ def arbitrary_recoil_line(self):
             self.arbitrary_nuclei, confirm = QInputDialog.getInt(self, 'Arbitrary Nuclei', 'Enter relative mass:')
             if not confirm:
                 return
-        self.toggle_recoil_line(self.actionArbitrary_nuclei, self.arbitrary_nuclei)
+        self.toggle_overplot_line(self.actionArbitrary_nuclei, self.arbitrary_nuclei, True)
+
+    def cif_file_powder_line(self):
+        cif_path = str(QFileDialog().getOpenFileName(self, 'Open CIF file', '/home', 'Files (*.cif)'))
+        key = path.basename(cif_path).rsplit('.')[0]
+        self.toggle_overplot_line(self.actionCIF_file, key, False, extra_info=cif_path)
```

Comment:
Because the key is the CIF file name, and this is asked even on an "uncheck" action, if the user does not select the same CIF file, `hide_overplot_line` does not know which line to remove. 

On the plus side, this lets users add powder lines from more than one CIF at a time; on the minus side, it's impossible to remove the lines without replotting the slices.

For the moment, I think if you just add a check for `self.actionCIF_file.isChecked()` and only if it it is checked to ask the user for a file, this would be fine. (E.g. it restricts the user to plotting single CIF file at a time... but we can change to code later if the ability to plot more than one CIF file is requested).